src/Sulu/Bundle/PageBundle/Tests/Functional/Controller/AdminControllerTest.php

```diff
@@ -163,15 +163,15 @@ public function testPagesFormMetadataAction()
         $this->assertObjectHasAttribute('url', $overviewType->form);
         $this->assertObjectHasAttribute('article', $overviewType->form);
         $this->assertObjectHasAttribute('schema', $overviewType);
-        $this->assertCount(1, (array) $overviewType->schema->properties);
+        $this->assertCount(5, (array) $overviewType->schema->properties);
         $this->assertEquals('array', $overviewType->schema->properties->block->type);
         $this->assertCount(2, $overviewType->schema->properties->block->items->anyOf);
         $this->assertEquals(['title', 'type'], $overviewType->schema->properties->block->items->anyOf[0]->required);
-        $this->assertCount(1, (array) $overviewType->schema->properties->block->items->anyOf[0]->properties);
+        $this->assertCount(3, (array) $overviewType->schema->properties->block->items->anyOf[0]->properties);
         $this->assertEquals('type', $overviewType->schema->properties->block->items->anyOf[0]->properties->type->name);
         $this->assertEquals('type1', $overviewType->schema->properties->block->items->anyOf[0]->properties->type->const);
         $this->assertEquals(['image', 'type'], $overviewType->schema->properties->block->items->anyOf[1]->required);
-        $this->assertCount(1, (array) $overviewType->schema->properties->block->items->anyOf[1]->properties);
+        $this->assertCount(3, (array) $overviewType->schema->properties->block->items->anyOf[1]->properties);
```

Comment:
The `SchemaMetadata` filters out every property without a schema. And because now every `text_line` property has a schema, it's not filtered out anymore.

Reply:
Ah, ok... I think these tests are not well written, because it is pretty much unclear what's happening here :see_no_evil: But it is fine for now.